example/src/Demo.jsx

```diff
@@ -31,6 +31,8 @@ function Demo({ fetchSwagger, status, docs, oas, oauth }) {
               'api-setting': Object.assign(extensions.defaults, oas),
             }}
             baseUrl={'/'}
+            // Uncomment this if you want to test enterprise-structured URLs
+            // baseUrl={'/child/v1.0'}
```

Comment:
If baseUrl has a slash on the end of it, then the resulting URL will have a double slash:

```
http://localhost:9966/child/v1.0//docs/link
```

I think this is okay, as long as we can guarantee that ReadMe will only ever pass in baseUrls with no slash on the end.